.github/workflows/ci.yml

```diff
@@ -77,31 +76,30 @@ jobs:
           key: ${{ runner.os }}-${{ matrix.WINDOWS.ARCH }}-${{ steps.setup-python.outputs.python-version }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
       - run: pip install tox
-      - run: tox
+      - run: tox --discover $(which python)
         env:
-          TOXENV: ${{ matrix.PYTHON.TOXENV }}
           CARGO_TARGET_DIR: ${{ format('{0}/src/_bcrypt/target/', github.workspace) }}
   linux:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
         PYTHON:
-          - {VERSION: "3.13", TOXENV: "pep8,packaging"}
-          - {VERSION: "3.13", TOXENV: "mypy"}
```

Comment:
Dropping these changes what the behavior is.

Reply:
oops! thanks...